### PR TITLE
behnaz/newfundmethod-nganluong

### PR DESCRIPTION
**Pull request by behnaz-fs**

### CODE REVIEW

File: email/templates/marketing/ngan-fundmethod.html [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+                        <!-- button -->
+                        <tr>
+                            <td bgcolor="#ffffff" align="center" style="padding: 0px 30px 0px; color: #333333; font-family: 'IBM Plex Sans', Arial, sans-serif; font-size: 16px; font-weight: 400; line-height: 24px;">
+                                <!--[if (gte mso 9)|(IE)]>
+                                    <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="https://deriv.com/payment-methods/nganluong-payment-method.pdf" style="height:40px;v-text-anchor:middle;width:220px;" arcsize="8%" stroke="f" fillcolor="#ff444f">
+                                    <w:anchorlock/>
+                                    <center style="color:#ffffff;font-family:sans-serif;font-size:16px;font-weight:bold;">Download PDF</center>
+                                    </v:roundrect>
+                                <![endif]-->
+                                <a class="button-secondary" href="https://deriv.com/payment-methods/nganluong-payment-method.pdf" style="mso-hide:all;"><span>Download PDF</span></a>
```
Comment (sash-binary) [inline email/templates/marketing/ngan-fundmethod.html:243]: remove span from link 
```suggestion
                                <a class="button-secondary" href="https://deriv.com/payment-methods/nganluong-payment-method.pdf" style="mso-hide:all;">Download PDF</a>
```

File: email/templates/marketing/ngan-fundmethod.html [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+        <title>Welcome</title>
+        <!--[if mso]>
+            <style type="text/css">
+                * {font-family: Arial, sans-serif !important;}
+                table {border-collapse: collapse;}
+                a {color: #ff444f; font-family: Arial, sans-serif;}
+            </style>
+        <![endif]-->
+        <style type="text/css">
+            /* http://meyerweb.com/eric/tools/css/reset/
```
Comment (sash-binary) [inline email/templates/marketing/ngan-fundmethod.html:18]: no need for html5 reset

**Review by sash-binary — CHANGES_REQUESTED** [review]

File: email/templates/marketing/ngan-fundmethod.html (hunk 1 above)
Comment (behnaz-fs) [reply email/templates/marketing/ngan-fundmethod.html:243]: Done

File: email/templates/marketing/ngan-fundmethod.html (hunk 2 above)
Comment (behnaz-fs) [reply email/templates/marketing/ngan-fundmethod.html:18]: css reset helps to remove default browsers setting e.g padding/margin, then set padding/margin to be sure have same result in all browsers, right?

Comment (sash-binary) [reply email/templates/marketing/ngan-fundmethod.html:18]: email is not shown in browser, so no need

Comment (behnaz-fs) [reply email/templates/marketing/ngan-fundmethod.html:18]: it helps to have same setting, so help to keep consistency

**Pull request merged**